effekt/shared/src/main/scala/effekt/Typer.scala

```diff
@@ -57,7 +57,7 @@ object Typer extends Phase[NameResolved, Typechecked] {
               // bring builtins into scope
               builtins.rootTerms.values.foreach {
                 case term: BlockParam =>
-                  Context.bind(term, term.tpe)
+                  Context.bind(term, term.tpe.get)
```

Comment:
I guess this is safe because we assume builtins are always annotated with their types, right?

Maybe document like:
```suggestion
                  Context.bind(term, term.tpe.getOrElse { INTERNAL_ERROR("Builtins should always be annotated with their types.") })
```